src/encoded/static/components/dataset.js

```diff
@@ -56,6 +56,7 @@ class AnnotationComponent extends React.Component {
         const context = this.props.context;
         const itemClass = globals.itemClass(context, 'view-item');
         const adminUser = !!(this.context.session_properties && this.context.session_properties.admin);
+        const experimentsUrl = `/search/?type=Experiment&possible_controls.accession=${context.accession}`;
```

Comment:
It is not a big deal. It may be a good idea to move this to a class-wide variable. Not just to follow the D.R.Y principle, but to make it  easier to identify this url path in case it changes and it needs to be witch-hunted.